### ISSUE TITLE
add an "embed" link

### ISSUE THREAD

**Issue by stchang** [vishesh/whalebin#6]
This is the embedding feature as discussed with Matthias. This should basically just show the iframe code I think?

**Comment by vishesh** [comment]
Yeah, just iframe  would work. What would be nicer though is if we could know what width and height of iframe should be, depending upon game.
